app_dart/lib/src/service/config.dart

```diff
@@ -326,10 +326,15 @@ class Config {
 
   Future<String> generateGithubToken(gh.RepositorySlug slug) async {
     // GitHub's secondary rate limits are run into very frequently when making auth tokens.
+    const RetryOptions retryOptions = RetryOptions(maxAttempts: 3, delayFactor: Duration(seconds: 3));
```

Comment:
This should be injectable so tests can set `Duration.zero`, otherwise we're causing artificial delays in the tests